implementation/src/test/resources/META-INF/services/org.eclipse.microprofile.rest.client.spi.RestClientBuilderResolver

```diff
@@ -0,0 +1 @@
+io.smallrye.restclient.BuilderResolver
```

Comment:
why was that not needed before and is needed now?

Reply:
Because of `RestClientBuilderResteasyPropertiesTest` unit test.